pybamm/models/submodels/electrode.py

```diff
@@ -0,0 +1,73 @@
+#
+# Equation classes for the electrode
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pybamm
+
+
+class Ohm(pybamm.BaseModel):
+    """Ohm's law + conservation of current for the current in the electrodes.
+
+    Parameters
+    ----------
+    phi : :class:`pybamm.Symbol`
+        The electric potential in the electrodes ("electrode potential")
+    epsilon : :class:`pybamm.Symbol`
+        The (electrolyte/liquid phase) porosity (can be Variable or Parameter)
+    j : :class:`pybamm.Symbol`
+        An expression tree that represents the interfacial current density at the
+        electrode-electrolyte interface
+    param : parameter class
+        The parameters to use for this submodel
+
+    *Extends:* :class:`BaseModel`
+    """
+
+    def __init__(self, phi_s, eps, j, param):
+        super().__init__()
+
+        current = pybamm.standard_parameters.current_with_time
+
+        # algebraic model only
+        self.rhs = {}
+
+        # different bounday conditions in each electrode
+        if phi_s.domain == ["negative electrode"]:
+            i_s_n = -param.sigma_n * (1 - eps) ** param.b * pybamm.grad(phi_s)
+            self.algebraic = {phi_s: pybamm.div(i_s_n) + j}
+            self.boundary_conditions = {phi_s: {"left": 0}, i_s_n: {"right": 0}}
+            self.initial_conditions = {phi_s: 0}
+            self.variables = {
+                "Negative electrode solid potential": phi_s,
+                "Negative electrode solid current": i_s_n,
+            }
+        elif phi_s.domain == ["positive electrode"]:
+            i_s_p = -param.sigma_p * (1 - eps) ** param.b * pybamm.grad(phi_s)
+            self.algebraic = {phi_s: pybamm.div(i_s_p) + j}
+            self.boundary_conditions = {i_s_p: {"left": 0, "right": current}}
+            self.initial_conditions = {
```

Comment:
It probably fine for now, but I'm not sure whether we should put initial conditions for the space only ODEs since their initial state should really be found by their consistency with the initial conditions for c etc. I can't think of a better place to put this initial guess though.